PagingSample/app/src/main/java/paging/android/example/com/pagingsample/CheeseViewHolder.kt

```diff
@@ -16,27 +16,28 @@
 
 package paging.android.example.com.pagingsample
 
-import androidx.recyclerview.widget.RecyclerView
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
 
 /**
  * A simple ViewHolder that can bind a Cheese item. It also accepts null items since the data may
```

Comment:
CheeseListItem

Reply:
Still need to fix this comment, either "can bind a Cheese or Separator item" or "can bind a CheeseListItem"

Reply:
Done